components/atom/switch/src/index.scss

```diff
@@ -0,0 +1,116 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$sz-atom_switch-input-height: $sz-icon-l !default;
+$sz-atom_switch-width_comp_container: 34px !default;
+$sz-atom_switch-height_comp_container: 22px !default;
+
+$sz-atom_switch-input-height--large: $sz-icon-xl !default;
+$sz-atom_switch-width_comp_container--large: 49px !default;
+$sz-atom_switch-height_comp_container--large: 32px !default;
+
+$c-atom-switch-active: #60acd0 !default;
+$c-atom-switch-default: #dcdcdc !default;
+
+.sui-AtomSwitch {
+  display: inline-block;
+  cursor: pointer;
+
+  &.toggle-type {
+    & > div.container {
+      margin-top: $p-l;
+    }
+  }
+
+  &.single-type {
+    & > div.container > div.input-container {
+      margin-left: $m-l;
+    }
+  }
+
+  & > .container {
+    display: flex;
+    justify-content: center;
+    height: $sz-atom_switch-input-height;
+    padding-top: ($sz-atom_switch-input-height - $fz-m) / 4;
+
+    & > span.text {
```

Comment:
You should try to avoid descendant selectors. Moreover, them seem to be unnecessary most of the time.